yarn.lock

```diff
@@ -9906,6 +9911,11 @@
   "resolved" "https://registry.npmjs.org/uuid/-/uuid-8.3.2.tgz"
   "version" "8.3.2"
 
+"uuid@^9.0.0":
```

Comment:
Optional :
- [ ] No need to generate an id as it is a primary key and rails will add it automatically

Reply:
@hammaazarok Removed uuid from package file